cmd/kube-scheduler/app/options/options_test.go

```diff
@@ -329,6 +340,75 @@ profiles:
 				},
 			},
 		},
+		{
+			name: "v1beta1 config file",
+			options: &Options{
+				ConfigFile: v1beta1VersionConfig,
+				ComponentConfig: func() kubeschedulerconfig.KubeSchedulerConfiguration {
+					cfg, err := newDefaultComponentConfig()
```

Comment:
newDefaultComponentConfig was changed above to switch to using v1beta2 defaults. those happen to be identical to v1beta1 defaults, but if they changed, this test would pass when it should fail (loading a v1beta1 config should keep v1beta1 defaults)

Reply:
Done, copied newDefaultComponentConfig to newV1beta1DefaultComponentConfig for test purposes.